rest_api/sawtooth_rest_api/route_handlers.py

```diff
@@ -245,19 +251,24 @@ def __init__(self, stream_url, timeout=DEFAULT_TIMEOUT):
             data: JSON array of fully expanded Block objects
             head: The head used for this query (most recent if unspecified)
             link: The link to this exact query, including head block
+            paging: Paging info and nav, like total resources and a next link
         """
-        head = request.url.query.get('head', None)
-        ids = self._get_filter_ids(request)
+        paging_controls = self._get_paging_controls(request)
+        validator_query = client_pb2.ClientBlockListRequest(
+            head_id=request.url.query.get('head', None),
+            block_ids=self._get_filter_ids(request),
+            paging=self._make_proto_paging(paging_controls))
```

Comment:
suggest renaming this to `_make_paging_message`; less tied to protobuf as a name.